src/main/java/com/launchacademy/reviews/controllers/HomeController.java

```diff
@@ -4,7 +4,7 @@
 
 @Controller
 public class HomeController {
-  @GetMapping(value = {"/genres", "/genres/{id}", "/films/{id}", "/films/new"})
+  @GetMapping(value = {"/genres", "/genres/{id}", "/films/{id}", "/films/new", "/films/{id}/delete"})
```

Comment:
Since we have deleted the `DeleteFilm` component, we can remove the last path here `/films/{id}/delete`